### ISSUE TITLE
Related work

### ISSUE THREAD

**Issue by simi2525** [Tensor-Reloaded/Backforward-Propagation#2]
- Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift
	- TLDR ???
	- Should be cited
	- https://arxiv.org/abs/1502.03167
- Revisiting Internal Covariate Shift for Batch Normalization
	- TLDR ???
	- Should be cited
	- https://ieeexplore.ieee.org/abstract/document/9238401
- How Does Batch Normalization Help Optimization?
	- TLDR ???
	- Should be cited
	- https://proceedings.neurips.cc/paper/2018/hash/905056c1ac1dad141560467e0a99e1cf-Abstract.html
- Improving robustness against common corruptions by covariate shift adaptation
	- TLDR ???
	- Should be cited
	- https://proceedings.neurips.cc/paper/2020/hash/85690f81aadc1749175c187784afc9ee-Abstract.html
- An Internal Covariate Shift Bounding Algorithm for Deep Neural Networks by Unitizing Layers' Outputs
	- TLDR ??? (Seems they found a way to measure ICS and ICS bounds, and propose a parametric way of reducing it, we should try it)
	- Should be cited
	- https://arxiv.org/abs/2001.02814
